edu.cuny.citytech.refactoring.common.eval/src/edu/cuny/citytech/refactoring/common/eval/handlers/EvaluateRefactoringHandler.java

```diff
@@ -0,0 +1,28 @@
+package edu.cuny.citytech.refactoring.common.eval.handlers;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+
+/**
+ * Our abstract handler extends AbstractHandler, an IHandler base class.
+ *
+ * @see org.eclipse.core.commands.IHandler
+ * @see org.eclipse.core.commands.AbstractHandler
+ */
+public abstract class EvaluateRefactoringHandler extends AbstractHandler {
+
+	private static final boolean BUILD_WORKSPACE = false;
+	
+	private static CSVPrinter createCSVPrinter(String fileName, String[] header) throws IOException {
+		return new CSVPrinter(new FileWriter(fileName, true), CSVFormat.EXCEL.withHeader(header));
+	}
+
+	@Override
+	public abstract Object execute(ExecutionEvent event) throws ExecutionException;
```

Comment:
What is the purpose of this?

Reply:
This can probably be gotten rid of, in retrospect, was just thinking it might be useful to have all the extending classes in our project hook their overwritten methods to our abstract base class as opposed to the framework dependency ? IDK maybe this makes no sense.

Reply:
I think it should be fine if they want to just use the inherited version (whatever that does).